src/libmongoc/src/mongoc/mongoc-util.c

```diff
@@ -1025,3 +1032,62 @@ bin_to_hex(const uint8_t *bin, size_t bin_len)
 
    return out;
 }
+
+#ifdef _WIN32
+
+typedef NTSTATUS(APIENTRY *RTLVERIFYVERSIONINFO_FN)(PRTL_OSVERSIONINFOEXW VersionInfo,
+                                                    ULONG TypeMask,
+                                                    ULONGLONG ConditionMask);
+
+bool
+_mongoc_verify_windows_version(int major_version, int minor_version, int build_number, bool strictly_equal)
+{
+   static RTLVERIFYVERSIONINFO_FN pRtlVerifyVersionInfo;
+   OSVERSIONINFOEXW osvi;
+   bool matched;
+   int op = VER_GREATER_EQUAL;
```

Comment:
```suggestion
   BYTE op = VER_GREATER_EQUAL;
```

To match type expected by [VER_SET_CONDITION](https://learn.microsoft.com/en-us/windows/win32/api/winnt/nf-winnt-ver_set_condition)